src/Controls/src/Core/Platform/Android/BottomNavigationViewUtils.cs

```diff
@@ -73,10 +73,7 @@ internal static void UpdateEnabled(bool tabEnabled, IMenuItem menuItem)
 		{
 			Context context = mauiContext.Context;
 
-			while (items.Count < menu.Size())
-			{
-				menu.RemoveItem(menu.GetItem(menu.Size() - 1).ItemId);
-			}
+			menu.Clear();
```

Comment:
Need to verify this change against
https://github.com/dotnet/maui/commit/16a3972f6941a174d2df7b385613bda2955e6e66#diff-1787c6c4f33d04c0dd8bceddf563be45c136658b564e5a6be28161f37b79adc5R75-R80

Where I originally modified this. 

Make sure the issue fixed by that PR doesn't resurface here

Reply:
I think the idea is that we don't want to completely clear the menu and then re-add because then it causes a shuffling to happen visually
So, the point of walking backwards was to just sync the list, instead of just clear and re-add
I think will still need the old code but a variation of it

Reply:
@PureWeen I've added a commit. Let me know what you think